### ISSUE TITLE
Validate that the page has completely loaded before tests are run

### ISSUE THREAD

**Issue by mgifford** [mgifford/open-scans#178]
We should be able to check that and run the accessibility tests only after everything has loaded. Some pages are very slow and use a lot of JS to modify the DOM.
